platforms/shared/configuration/roles/setup/certmanager/tasks/create_mapping.yaml

```diff
@@ -0,0 +1,45 @@
+---
+##create mapping for acme
+- name: create Mapping for Acme
+  kubernetes.core.k8s:
```

Comment:
see if it makes sense to use move the service mapping and cluster issuer as helm chart